src/test/compile-fail/resolve-conflict-item-vs-extern-crate.rs

```diff
@@ -8,7 +8,8 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-fn std() {}    //~ ERROR the name `std` conflicts with an external crate
+fn std() {}
```

Comment:
`fn std` no longer conflicts since it defines `std` in the value namespace but the extern crate defines `std` in the type namespace.

Right now, `fn std` would not does with `mod std { ... }` for the same reason and I see no reason to treat `extern crate std` differently.